src/couch_replicator.hrl

```diff
@@ -25,3 +25,15 @@
 }).
 
 -type rep_id() :: {string(), string()}.
+-type db_doc_id() :: {binary(), binary() | '_'}.
+-type sec() :: non_neg_integer().
```

Comment:
Those types are shared between doc processor and worker. `sec()` is an alias for time (seconds) it just for readability as it has the units it the type, as time can be minutes, milliseconds etc.   `_` is make dialyzer happy, because we are building an ets match pattern which has DocId part as `_` when we remove all replications when a db shard goes away.